css/main.css

```diff
@@ -0,0 +1,457 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+#logo {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.menu-bars {
+  background: transparent;
+  border: none;
+  outline: none;
+}
+
+.menu-bars i {
+  width: 18px;
+  height: 12px;
+  color: white;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+}
+
+.intro,
+.about-me {
+  background-color: #1c1a19;
+  color: white;
+  position: relative;
+}
+
+.intro {
+  padding: 10%;
+  padding-bottom: 20vh;
+}
+
+.intro-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.intro-greet,
+.about-me h2 {
+  color: #ff6b00;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+.intro-me {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  align-items: center;
+  color: #fff5e1;
+}
+
+.intro-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.social {
+  display: flex;
+}
+
+.social-link {
+  padding: 5px;
+  background-color: transparent;
+  transition: 350ms;
+}
+
+.social-link:hover {
+  background-color: rgba(255, 255, 255, 0.096);
+}
+
+.social-icon {
+  height: 20px;
+}
+
+.about-me-section {
+  position: relative;
+  padding: 10%;
+  padding-bottom: 20vh;
+}
+
+.projects {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #e5e5e5;
+  padding: 10%;
+}
+
+.projects h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+}
+
+.projects hr {
+  margin: 25px 0;
+  border-color: #212121;
+}
+
+.first-card-description,
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  min-height: 200px;
+  margin: 20px;
+}
+
+.first-card-img {
+  width: 85vw;
+  height: auto;
+}
+
+.description,
+.white-description {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.blue-color,
+.blue-color * {
+  color: #091e42 !important;
```

Comment:
- [ ] [Optional] Would it be better to avoid using `!important` in your CSS ? 🤔
I quote from Mozilla Docs

> Using !important, however, is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets. 💣

For more infromation [Mozilla Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception)